pkg/cmd/server/api/types.go

```diff
@@ -417,6 +417,11 @@ type ImagePolicyConfig struct {
 	// MaxScheduledImageImportsPerMinute is the maximum number of image streams that will be imported in the background per minute.
 	// The default value is 60. Set to -1 for unlimited.
 	MaxScheduledImageImportsPerMinute int
+	// AllowedRegistriesForImport is a white list of registries that we allow to import the
+	// images from. By default docker.io, Red Hat registry, gcr.io, Amazon ECS, quay.io are
+	// allowed. If you want to import images from your private Docker Registry, you have to
+	// add it into the list.
+	AllowedRegistriesForImport []string
```

Comment:
@smarterclayton i this we also need wildcard (*) for registries like ECS. Does it make sense?